### PR TITLE
Expose Contact Center APIs to Graph V1.0

### PR DESCRIPTION
**Pull request by jackry**

### CODE REVIEW

File: api-reference/v1.0/api/audioroutinggroup-delete.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+## Response
+If successful, this method returns `204 No Content` response code. It does not return anything in the response body.
+
+## Example
+
+##### Request
+The following example shows the request.
+
+
+# [HTTP](#tab/http)
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/audioroutinggroup-delete.md:48]: GLOBAL: Please apply this change to all API topics.

These tabs get added via the weekly SDK snippet automation that runs on the main branch. In this case, you need to remove the corresponding SDK snippet links and HTTP H1 headings.

File: api-reference/beta/resources/audioroutinggroup.md [hunk 2]
```diff
@@ -21,7 +21,8 @@ The audio routing group stores a private audio route between participants in a m
 
 | Method                                                  | Return Type                               | Description                                  |
 |:--------------------------------------------------------|:------------------------------------------|:---------------------------------------------|
-| [Get audioRoutingGroup](../api/audioroutinggroup-get.md)| [audioRoutingGroup](audioroutinggroup.md) | Read properties and relationships of audioRoutingGroup object.|
+| [Create](../api/call-post-audioroutinggroups.md) | [audioRoutingGroup](audioroutinggroup.md)                   | Create a new audioRoutingGroup by posting to the audioRoutingGroups collection. |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/audioroutinggroup.md:24]: For simplicity and consistency, please remove "by posting to the audioRoutingGroups collection." Also, we apply bold formatting to resource names; please update the formatting of audioRoutingGroup.

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/v1.0/resources/audioroutinggroup.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+## Properties
+
+| Property      | Type              | Description                                                          |
+| :----------   | :---------------- | :--------------------------------------------------------------------|
+| id            | string            | Read-only.                                                           |
+| receivers     | collection(string) | List of receiving participant ids.                                   |
+| routingMode   | string            | Routing group mode.  Possible values are: `oneToOne`, `multicast`.   |
+| sources       | collection(string) | List of source participant ids.                                      |
+
+> **Note:** Routing mode determines the restrictions on the sources and receivers. Only the following routing groups are supported.
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/audioroutinggroup.md:33]: Please, 

- add "The" before "routing mode"
- replace the period with colon. Suggested revision: "...routing groups are supported:"
- add "The" before "receivers list"

File: api-reference/v1.0/resources/audioroutinggroup.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Namespace: microsoft.graph
+
+The audio routing group stores a private audio route between participants in a multiparty conversation. Source is the participant itself and the receivers are a subset of other participants in the multiparty conversation.
+
+## Methods
+
+| Method                                                  | Return Type                               | Description                                  |
+|:--------------------------------------------------------|:------------------------------------------|:---------------------------------------------|
+| [Get](../api/audioroutinggroup-get.md)| [audioRoutingGroup](audioroutinggroup.md) | Read properties and relationships of audioRoutingGroup object.|
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/audioroutinggroup.md:20]: We apply bold formatting to resource names; please update the formatting of audioRoutingGroup.

File: api-reference/v1.0/resources/audioroutinggroup.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+| [Update](../api/audioroutinggroup-update.md)            | [audioRoutingGroup](audioroutinggroup.md) | Update receivers list.                       |
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/audioroutinggroup.md:21]: For clarity, please add "the" before "receivers", and replace "receivers" with "receiver's".

File: api-reference/v1.0/resources/audioroutinggroup.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+| [Delete](../api/audioroutinggroup-delete.md)            | None                                      | Delete the audio routing group.              |
+
+## Properties
+
+| Property      | Type              | Description                                                          |
+| :----------   | :---------------- | :--------------------------------------------------------------------|
+| id            | string            | Read-only.                                                           |
+| receivers     | collection(string) | List of receiving participant ids.                                   |
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/audioroutinggroup.md:29]: In regular text, please use ID as the abbreviation for identity. Please, apply this change also to row 31.

File: api-reference/v1.0/resources/audioroutinggroup.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+| sources       | collection(string) | List of source participant ids.                                      |
+
+> **Note:** Routing mode determines the restrictions on the sources and receivers. Only the following routing groups are supported.
+> - `oneToOne` - sources and receivers have only one participant each.
+> - `multicast` - source has one participant but there are multiple receivers. Receivers list may be updated.
+
+> **Note:** If you create many audio routing groups (e.g. a bot per participant), only the audio of the top 4 dominant speakers is forwarded. It means even with customized audio routing group, if the speaker is not loud enough in the main mixer, he/she cannot be heard by the bot even if there is a private audio group just for this speaker and the bot.
+
+## Relationships
+None
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/audioroutinggroup.md:40]: For consistency, please add a period at the end of the sentence.

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: changelog/Microsoft.Skype.Calling.json [hunk 8]
```diff
@@ -1,5 +1,49 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the [audioRoutingGroup](https://docs.microsoft.com/en-us/graph/api/resources/audioRoutingGroup?view=graph-rest-1.0) resource",
+          "Target": "audioRoutingGroup"
+        },
+        {
+          "Id": "f6456ed2-2c6e-4c76-9d55-36f901c69ebf",
+          "ApiChange": "Resource",
+          "ChangedApiName": "audioRoutingGroups",
+          "ChangeType": "Addition",
+          "Description": "Added the **audioRoutingGroups** relationship to [call](https://docs.microsoft.com/en-us/graph/api/resources/call?view=graph-rest-1.0) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Skype.Calling.json:36]: The audioRoutingGroups (create and list methods?) are not added to the call resource topic as a relationship. Rather, the create and list methods were added to the Methods table. Is that correct?

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/incomingCallOptions.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+# incomingCallOptions resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+A class that contains the options for an incoming call. This is inherited from [callOptions](callOptions.md).
+
+## Properties
```
Comment (JarbasHorst) [inline api-reference/beta/resources/incomingCallOptions.md:18]: Should the Properties table have some properties?

File: api-reference/beta/resources/invitationparticipantinfo.md [hunk 10]
```diff
@@ -22,6 +22,9 @@ Represents an entity that is being invited to a group call.
 | endpointType                       | String                        | The type of endpoint. Possible values are: `default`, `voicemail`. |
 | identity                           | [identitySet](identityset.md) | The [identitySet](identityset.md) associated with this invitation.                   |
 | replacesCallId                     | String                        | Optional. The call which the target idenity is currently a part of. This call will be dropped once the participant is added. |
+| removeFromDefaultAudioRoutingGroup | Boolean                       | Remove self participant from default audio group. |
+| hidden                             | Boolean                       | Hide in the roster. |
+| participantId                      | String                        | The participant id. |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/invitationparticipantinfo.md:27]: In regular text, please use ID as the abbreviation for identity.

Please, apply this change also here: api-reference/v1.0/resources/invitationparticipantinfo.md

File: api-reference/beta/resources/outgoingCallOptions.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
+---
+title: "outgoingCallOptions resource type"
+description: "A class that contains the options for an outgoing call"
```
Comment (JarbasHorst) [inline api-reference/beta/resources/outgoingCallOptions.md:3]: Please apply these changes also here: api-reference/v1.0/resources/outgoingCallOptions.md 

For consistency, please start the description with "Represents...". Also, please add a period at the end of the sentence.

Please apply this change also to line 16.

File: api-reference/beta/resources/outgoingCallOptions.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+doc_type: resourcePageType
+---
+
+# outgoingCallOptions resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+A class that contains the options for an outgoing call. This is inherited from [callOptions](callOptions.md).
```
Comment (JarbasHorst) [inline api-reference/beta/resources/outgoingCallOptions.md:16]: Please apply these changes also here: api-reference/v1.0/resources/outgoingCallOptions.md 

For consistency, I recommend you to move the line starting with "This is..." to a new line. For simplicity, I recommend you to start the sentence with "Inherited". Suggested revision:

```
Represents a class that contains the options for an outgoing call.

Inherited from [callOptions](callOptions.md).
```

File: api-reference/beta/resources/outgoingCallOptions.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+## Properties
+|Property                 |Type                      |Description                                                                        |
+|:---                     |:---                      |:---                                                                               |
+|allowGuestToBypassLobby  |Boolean                   |The value indicating whether a bot with guest identity joins a group call should bypass lobby.    |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/outgoingCallOptions.md:21]: Please apply this change also here: api-reference/v1.0/resources/outgoingCallOptions.md 

Please, avoid using gerunds where possible; in this case, you can change "indicating" to "that indicates".

File: api-reference/beta/resources/callOptions.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
+---
+title: "callOptions resource type"
+description: "An abstract base class that contains the options for a call"
```
Comment (JarbasHorst) [inline api-reference/beta/resources/callOptions.md:3]: Please, add a period at the end of the sentence.

Please, apply this change also here: api-reference/v1.0/resources/callOptions.md

File: changelog/Microsoft.Skype.Calling.json [hunk 15]
```diff
@@ -1,5 +1,153 @@
...
+        }
+      ],
+      "Id": "7a82d297-d6f3-4fd1-8ddf-770db83d3637",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2021-11-18T06:24:28.2074626Z",
+      "WorkloadArea": "Cloud communications",
+      "SubArea": "Call"
+    },
+    {
```
Comment (JarbasHorst) [inline changelog/Microsoft.Skype.Calling.json:55]: Please, merge all v1.0 changelog records into one.

File: changelog/Microsoft.Skype.Calling.json [hunk 16]
```diff
@@ -1,5 +1,153 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "5a70b45d-ad69-46f8-a0d1-1f254b2d5f4e",
+          "ApiChange": "Property",
+          "ChangedApiName": "hideBotAfterEscalation",
+          "ChangeType": "Addition",
+          "Description": "Added the **hideBotAfterEscalation** property to [callOptions](https://docs.microsoft.com/en-us/graph/api/resources/callOptions?view=graph-rest-beta) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Skype.Calling.json:10]: Please, add "the" before the resource topic. For example, ...property to the [callOptions]...

Please apply this change to all new entries in the changelog.

File: changelog/Microsoft.Skype.Calling.json [hunk 17]
```diff
@@ -1,5 +1,153 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "5a70b45d-ad69-46f8-a0d1-1f254b2d5f4e",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Skype.Calling.json:6]: Please, add the following three new changelog entries to the beta changelog record:

New resources:
- incomingCallOptions
- outgoingCallOptions
- callOptions: This is a new resource should actually appear in the changelog and not the property "hideBotAfterEscalation"

File: changelog/Microsoft.Skype.Calling.json [hunk 18]
```diff
@@ -1,5 +1,153 @@
...
+      ],
+      "Id": "7a82d297-d6f3-4fd1-8ddf-770db83d3637",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2021-11-18T06:24:28.2074626Z",
+      "WorkloadArea": "Cloud communications",
+      "SubArea": "Call"
+    },
+    {
+      "ChangeList": [
```
Comment (JarbasHorst) [inline changelog/Microsoft.Skype.Calling.json:56]: Please, add the following three new changelog entries to the v1.0 changelog record:

New resources:
- incomingCallOptions
- outgoingCallOptions
- callOptions: This is a new resource should actually appear in the changelog and not the property "hideBotAfterEscalation"

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

**Comment by Lauragra** [comment]
@jackry , I'm closing this PR due to lack of activity/response. Some of the content included in this PR is already live/published so this looks to be out of date. If we still need this PR, please reopen and update per comments.

**Pull request closed without merging**